src/database/models/index.js

```diff
@@ -0,0 +1,34 @@
+import { readdirSync } from 'fs';
+import { basename as _basename, join } from 'path';
+import Sequelize, { DataTypes } from 'sequelize';
+const basename = _basename(__filename);
+const env = process.env.NODE_ENV || 'development';
+const config = require(__dirname + '/../config/config.js')[env];
+const db = {};
+
+let sequelize;
+if (config.use_env_variable) {
+  sequelize = new Sequelize(process.env[config.use_env_variable], config);
+} else {
+  sequelize = new Sequelize(config.database, config.username, config.password, config);
```

Comment:
This line of code `config.database, config.username, config.password,` should be replaced with `config.url`